docs/datasets/hls_tiling.ipynb

```diff
@@ -0,0 +1,777 @@
+{
```

Comment:
<blockquote>This notebook analyzes the performance benchmarks</blockquote><p> </p><p>Could you explain where the benchmark data is coming from?</p><p> </p><blockquote><strong>Collection choice</strong>: HLSS30 has a more frequent revisit rate than HLSL30</blockquote><p> </p><p>Could you clarify why this impacts tiling performance?</p><p> </p><p>The summarized recommendations is excellent! I think this will be very helpful.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/titiler-cmr/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>